osu.Game/Overlays/Settings/Sections/Maintenance/StableDirectoryLocationDialog.cs

```diff
@@ -6,13 +6,14 @@
 using osu.Framework.Graphics.Sprites;
 using osu.Framework.Screens;
 using osu.Game.Overlays.Dialog;
+using osu.Game.Screens;
 
 namespace osu.Game.Overlays.Settings.Sections.Maintenance
 {
     public class StableDirectoryLocationDialog : PopupDialog
     {
-        [Resolved]
-        private OsuGame game { get; set; }
+        [Resolved(canBeNull: true)]
+        private IPerformFromScreenRunner performer { get; set; }
```

Comment:
Where's this change of nullability coming from? The updated usage below is assuming that this is not null, too (no conditional access operator).

Reply:
Failed copy paste, oops.